.changeset/too-many-holidays.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-cloud-sdk/http-client': minor
+---
+
+[Fixed Issue] Fix the `executeHttpRequest` function, so it accepts the same parameters as in version 1.
+[Compatibility Note] Deprecate one overload of the `executeHttpRequest` function, that accepts `HttpRequestConfigWithOrigin` as a parameter. Use the new function `executeHttpRequestWithOrigin` as replacement.
```

Comment:
[q] Does it make sense to also add an entry for a new feature here: "executeHttpRequestWithOrigin" ?